scripts/zones/Windurst_Woods/npcs/Apururu.lua

```diff
@@ -122,19 +105,7 @@ function onEventFinish(player,csid,option)
     if (csid == 137) then
         player:setVar("MissionStatus",1);
 
-        player:addKeyItem(dsp.ki.FIRST_DARK_MANA_ORB);    -- Give the player the key items
-        player:addKeyItem(dsp.ki.SECOND_DARK_MANA_ORB);
-        player:addKeyItem(dsp.ki.THIRD_DARK_MANA_ORB);
-        player:addKeyItem(dsp.ki.FOURTH_DARK_MANA_ORB);
-        player:addKeyItem(dsp.ki.FIFTH_DARK_MANA_ORB);
-        player:addKeyItem(dsp.ki.SIXTH_DARK_MANA_ORB);
-
-        player:messageSpecial(KEYITEM_OBTAINED,dsp.ki.FIRST_DARK_MANA_ORB);    -- Display the key item messages
-        player:messageSpecial(KEYITEM_OBTAINED,dsp.ki.SECOND_DARK_MANA_ORB);
-        player:messageSpecial(KEYITEM_OBTAINED,dsp.ki.THIRD_DARK_MANA_ORB);
-        player:messageSpecial(KEYITEM_OBTAINED,dsp.ki.FOURTH_DARK_MANA_ORB);
-        player:messageSpecial(KEYITEM_OBTAINED,dsp.ki.FIFTH_DARK_MANA_ORB);
-        player:messageSpecial(KEYITEM_OBTAINED,dsp.ki.SIXTH_DARK_MANA_ORB);
+        npcUtil.giveKeyItem(player, {dsp.ki.FIRST_DARK_MANA_ORB, dsp.ki.SECOND_DARK_MANA_ORB, dsp.ki.THIRD_DARK_MANA_ORB, dsp.ki.FOURTH_DARK_MANA_ORB, dsp.ki.FIFTH_DARK_MANA_ORB, dsp.ki.SIXTH_DARK_MANA_ORB});
```

Comment:
wide line, mind breaking down? I'd rather up/down scroll than left right scroll, but you can format it in whatever way is aesthetically pleasing to you.